wazuh/wazuh
# Perform re-scan when cluster name changes

|Related issue|
|---|
| #23540 |

## Description
This PR aims to fix the inability to see vulnerabilities once the cluster name changes

We will keep track of the cluster name and in the case of a change, a scan will be performed over all agents

## Tests
Due to the nature of the changes we should test all variations of policy changes:
- Enable/disable VD
  - Manager policy
  - General policy
- Change the cluster name
- Enable/disable cluster
